### PR TITLE
Issue 9: create a setup.py

### PR DESCRIPTION
**Pull request by romain-dartigues**
Work on `setup.py`.

I think it is required to rename "app" to a more explicit name which will be installed as a Python library (docker_leash in this case).

### CODE REVIEW

**Comment by romain-dartigues** [comment]
## Notes

### Classifiers

https://pypi.python.org/pypi?%3Aaction=list_classifiers

### Link to a tarball

https://developer.github.com/v3/repos/contents/#get-archive-link

### README

Pypi seems to favor [ReST](http://docutils.sourceforge.net/rst.html) over Markdown ([SO](https://stackoverflow.com/a/26737258)); that's why I copied part of the README in setup.py.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+# stdlib
+import io
+import re
+
+# dependencies
+from setuptools import setup
+
+version = None
+with io.open('app/leash_server.py', 'rt', encoding='utf8') as fobj:
```
Comment (kumy) [inline setup.py]: Shouldn't it be `docker_leash` here too?

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+_github_path = 'docker-leash/leash-server'
+
+
+
+setup(
+    name='docker-leash',
+    version=version,
+    author='Mathieu Alorent',
+    author_email='docker-leash@kumy.org',
```
Comment (kumy) [inline setup.py]: It's '.net' not .org ;)

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        'docker_leash',
+    ],
+    package_data={
+        '': ['*.yml']
+    },
+    platforms='any',
+    py_modules=(
+    ),
+    install_requires=[
+        'Flask>=0.12',
```
Comment (kumy) [inline setup.py]: Does it imply to maintain dependencies here and in requirements.txt?

Comment (romain-dartigues) [reply setup.py]: Yes (or no).

To quote https://packaging.python.org/discussions/install-requires-vs-requirements/:
> Whereas install_requires defines the dependencies for a single project, Requirements Files are often used to define the requirements for a complete Python environment.

I think only the actual `requirements.dev.txt` is required, the other dependencies could be referenced in `setup.py`.

**Pull request merged**